tests/testthat/test_addLeadingZeros.R

```diff
@@ -1,6 +1,7 @@
 
 ### set up test data frames
-dat <- data.frame(v1 = c("0","300","e",NA),
+dat <- data.frame(ID=c("112324234244","112324234245","131224234246","112324234247"),
```

Comment:
LGTM

However, line 10 seems redundant to line 20 to me? And the test title in line 19 is really confusing to me?

Reply:
Thank you for pointing this out! This test was ill-labeled, sorry. I will improve it also by adding additional tests.

Reply:
LGTM